.eslintrc.json

```diff
@@ -8,6 +8,7 @@
   },
   "rules": {
     "no-console": "warn",
-    "quotes": ["error", "single"]
+    "@typescript-eslint/no-var-requires": "off", 
```

Comment:
Please, don't put `no-var-requires` off

Reply:
When I remove this can cause the ci/cd eslintsr errors, let me try to remove them and see what will be happen.

Reply:
Ok Done This was made It:

 "ignorePatterns": ["dist/**/*"]